build-aux/Jenkinsfile.full

```diff
@@ -278,7 +267,7 @@ pipeline {
       steps {
         timeout(time: 15, unit: "MINUTES") {
           sh (script: 'rm -rf apache-couchdb-*', label: 'Clean workspace of any previous release artifacts' )
-          sh "./configure --spidermonkey-version ${spidermonkey}"
+          sh "./configure"
```

Comment:
I think you can also remove the `environment` block above this; it was only there to supply `${spidermonkey}` for this command.

Reply:
Great catch, removed.